src/pages/denial_questions/DenialQuestions.page.tsx

```diff
@@ -161,76 +173,124 @@ export function DenialQuestions() {
+                              {option.label}
+                            </label>
+                          </div>
+                        ))
+                      ) : (
                         <input
-                          type="checkbox"
+                          type="text"
                           id={question.name}
                           name={question.name}
-                          value={option.value}
-                          checked={
-                            (question.name in answers
+                          value={String(
+                            question.name in answers
                               ? answers[question.name as keyof CreateDenialRequestOptions]
-                              : '') === option.value
-                          }
+                              : ''
+                          )}
                           onChange={handleAnswerChange}
-                          className="form-check-input"
+                          className="form-control"
                         />
-                        <label htmlFor={question.name} className="form-check-label">
-                          {option.label}
-                        </label>
-                      </div>
-                    ))
-                  ) : (
-                    <input
-                      type="text"
-                      id={question.name}
-                      name={question.name}
-                      value={String(
-                        question.name in answers
-                          ? answers[question.name as keyof CreateDenialRequestOptions]
-                          : ''
                       )}
-                      onChange={handleAnswerChange}
-                      className="form-control"
-                    />
+                    </>
+                  )}
+                  {!question.name && (
+                    // Optional: Render something for non-interactive questions
+                    <div></div>
                   )}
-                </>
-              )}
-              {!question.name && (
-                // Optional: Render something for non-interactive questions
-                <div></div>
-              )}
-            </div>
-          ))}
-          <button type="button" className="btn btn-green" onClick={handleNextQuestionSet}>
-            {currentQuestionSetIndex < questions.length - 1 ? 'Next' : 'Submit'}
-          </button>
+                </div>
+              )
+          )}
+          <Button
+            radius="md"
+            size="lg"
+            className={classes.primaryColor}
+            onClick={handleNextQuestionSet}
+            style={{ textTransform: 'uppercase' }}
+          >
+            {currentQuestionSetIndex === 0
+              ? 'Start'
+              : currentQuestionSetIndex < questions.length - 1
+                ? 'Next'
+                : 'Submit'}
+          </Button>
         </div>
```

Comment:
Gotcha so the plan here is to ask the initial set of questions then trigger a submit and fetch another set of questions? (Just asking since I don't see anything for the getting additional questions).